src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/events/ChangeMerged.java

```diff
@@ -80,11 +83,21 @@ public String getModifiedRef() {
         return null;
     }
 
+    /**
+     * @return ChangeMerge revision
+     */
+    public String getMergedRevision() {
```

Comment:
I would prefer the property has the same name _(in camelCase)_ as the json property.